_includes/footer.html

```diff
@@ -2,58 +2,55 @@
   <div class="container">
     <div class="row">
       <div class="col-sm-3">
-        <h5>Modules</h5>
+        <h5>Getting Started</h5>
         <ul class="list-unstyled">
-         <li><a href="/push/">AeroGear<strong>Push</strong></a></li>
-         <li><a href="/security/">AeroGear<strong>Security</strong></a></li>
-         <li><a href="/sync/">AeroGear<strong>Sync</strong></a></li>
-         <li><a href="/digger/">AeroGear<strong>Digger</strong></a></li>
+          <li><a href="/">Overview</a></li>
+          <li><a href="/">Installation &amp; Configuration</a></li>
+          <li><a href="/">Creating Mobile Apps</a></li>
+          <li><a href="/">Enabling Mobile Services</a></li>
         </ul>
       </div><!-- col -->
-      
+
       <div class="col-sm-3">
-        <h5>Platforms</h5>
+        <h5>Mobile Services</h5>
         <ul class="list-unstyled">
-          <li><a href="/android/">Android</a></li>
-          <li><a href="/ios/">iOS</a></li>
-          <li><a href="/cordova/">Cordova</a></li>
-          <li><a href="/javascript/">JavaScript</a></li>
-          <li><a href="/windows/">Windows</a></li>
+         <li><a href="/services/push/">AeroGear <strong>Push</strong></a></li>
+         <li><a href="/services/digger/">AeroGear <strong>Mobile CI/CD</strong></a></li>
+         <li><a href="/services/metrics/">AeroGear <strong>Metrics</strong></a></li>
+         <li><a href="/services/crc/">AeroGear <strong>CRC</strong></a></li>
+         <li><a href="/services/keycloak">AeroGear <strong>Keycloak</strong></a></li>
         </ul>
       </div><!-- col -->
-      
+
       <div class="col-sm-3">
-        <h5>Aerogear</h5>
+        <h5>Mobile SDKs</h5>
         <ul class="list-unstyled">
-          <li><a href="/getstarted/downloads">Downloads and snippets</a></li>
-          <li><a href="/getstarted/guides">Guides</a></li>
-          <li><a href="/getstarted/demos">Demos</a></li>
-          <li><a href="/docs/specs/">Docs</a></li>
-          <li><a href="/docs/planning/">Road Map</a></li>
+         <li><a href="/sdks/android/">Android</a></li>
+         <li><a href="/sdks/cordova/">Cordova</li>
+         <li><a href="/sdks/ios/">iOS</a></li>
+         <li><a href="/sdks/xamarin/">Xamarin</a></li>
         </ul>
       </div><!-- col -->
       
-      
       <div class="col-sm-3">
         <h5>COMMUNITY</h5>
         <ul class="list-unstyled">
-          <li><a href="https://github.com/aerogear/">GitHub</a></li>
-          <li><a href="/docs/planning/">Jira Repo Issues</a></li>
-          <li><a href="https://lists.jboss.org/mailman/listinfo/aerogear-dev">Developer mailing list</a></li>
-          <li><a href="https://lists.jboss.org/mailman/listinfo/aerogear-users">Users mailing list</a></li>
+          <li><a target="_blank" href="https://github.com/aerogear/">GitHub</a></li>
+          <li><a target="_blank" href="/docs/planning/">Jira Repo Issues</a></li>
```

Comment:
Is that the right link?

Reply:
I think so - but we will need to update the planning page with the proper links/urls

Reply:
https://issues.jboss.org/browse/AEROGEAR.

Reply:
oh ok cool